cfme/configure/access_control.py

```diff
@@ -285,17 +285,42 @@ def create(self):
              action=form_buttons.add)
         flash.assert_success_message('Group "{}" was saved'.format(self.description))
 
-    def retrieve_ldap_user_groups(self):
+    def _retrieve_ldap_user_groups(self):
         sel.force_navigate('cfg_accesscontrol_group_add')
         fill(self.group_form, {'lookup_ldap_groups_chk': True,
                                'user_to_look_up': self.user_to_lookup,
                                'username': self.ldap_credentials.principal,
                                'password': self.ldap_credentials.secret,
                                },
-             action=form_buttons.retrieve)
+             )
+        # adding sleep due to JS issue
+        import time
+        time.sleep(3)
+        sel.click(form_buttons.retrieve)
```

Comment:
In this case, you might like to use sel.wait_for_element instead of sleep, that might work here.